examples/IBM.Watson.Assistant.v2.Examples/IBM.Watson.Assistant.v2.Examples.csproj

```diff
@@ -6,11 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="IBM.Cloud.SDK.Core" Version="1.0.0-rc9" />
+    <PackageReference Include="IBM.Cloud.SDK.Core" Version="1.0.0-rc91" />

```

Comment:
This version number seems like it really jumped 😂

Reply:
lol yeah nuget didn't like `rc10`... in hindsight i should have started with `rc01`